h2o-py/h2o/explanation/_explain.py

```diff
@@ -1230,6 +1230,49 @@ def pd_multi_plot(
 
 def _center(col):
     col[:] = col - col[0]
+def _prepare_grouping_frames(frame, grouping_variable):
+    if (grouping_variable not in frame.names):
+        raise ValueError("Grouping variable '" + grouping_variable + "' is not present in frame!")
+    if (not frame[grouping_variable].isfactor()[0]):
+        raise ValueError("Grouping variable has to be categorical!")
+    categories = frame[grouping_variable].categories()
+    if len(categories) > 10:
+        raise ValueError("Grouping variable option is supported only for variables with 10 or fewer levels!")
```

Comment:
`10` looks like a magic number, is there reasoning for this? Should it be also configurable somehow?

Reply:
it's a requirement specification, it shouldn't be configurable.